src/Modal.php

```diff
@@ -337,21 +265,9 @@ protected function renderView(): void
             $this->template->del('ActionContainer');
         }
 
-        // call modal creation first
-        if (isset($this->options['modal_option'])) {
-            $this->js(true)->modal($this->options['modal_option']);
-        } else {
-            $this->js(true)->modal();
-        }
-
-        // add setting if available.
-        if (isset($this->options['setting'])) {
-            foreach ($this->options['setting'] as $key => $value) {
-                $this->js(true)->modal('setting', $key, $value);
```

Comment:
@ibelar any idea why the settings were not passed to the Fomantic-UI Modal constructor (initial `->js(true)->modal` call) directly?